frontend/src/components/account/AccountPage.tsx

```diff
@@ -361,8 +361,8 @@ export default function AccountPage({
             return (
               <>
                 {i.value.length > 0 && <div className={classes.subtitle}>{i.name}:</div>}
-
-                <SectorsPreview hubs={i.value} />
+                {/* TODO: SectorsPreview accepts sectors and no hubs  */}
+                <SectorsPreview hubs={i.value} /> {/* TODO: fix this*/}
```

Comment:
this is currently broken due to the previous branch having changed it.
@fateme-ramezanpour do you want to fix it in this PR or in a later one?